src/app/SharpRaven/Data/SentryStacktrace.cs

```diff
@@ -91,7 +84,7 @@ public override string ToString()
 
             StringBuilder sb = new StringBuilder();
 
-            foreach (var frame in Frames)
+            foreach (var frame in Frames.Reverse())
```

Comment:
Ah, I see. I remember writing that test, thinking it would be valuable having Sentry's object model directly represent and reflect that of `System.Exception`. Ideas on how to make that test more i18n friendly?